src/test/fakeCurrency.js

```diff
@@ -49,6 +50,10 @@ class FakeCurrencyEngine {
       createReaction(state => state.blockHeight, onBlockHeightChanged)
     )
 
+    this.MAX_SPENDABLE_SHARE = 0.9 + 0.1 * Math.random()
```

Comment:
Wouldn't it be better to make the test deterministic, loading different balances into the fake wallet and asserting that the `getMaxSpendable` returns the right value?

You can use `store.dispatch({ type: 'SET_BALANCE', payload: 20 })` to change the balance. There are examples of how to do this all over the `currencyWallet.test.js` file.

Reply:
@koretskiyav What's the status on this PR? I see you have another branch `get_max_pendable` that seems to be newer but there's no PR for it.